pkg/hal/cli/component/push.go

```diff
@@ -28,8 +28,8 @@ type pushOptions struct {
 }
 
 var (
-	pushExample = ktemplates.Examples(`  # Deploy the components client-sb, backend-sb
-  %[1]s -c client-sb,backend-sb`)
+	pushExample = ktemplates.Examples(`  # Deploy the 'foo' component
```

Comment:
Why is this change in this PR?

Reply:
Oups, I have slipt in this commit. I'll revert it and open another PR